crates/ruff_python_ast/src/nodes.rs

```diff
@@ -3176,6 +3176,63 @@ pub struct Decorator {
     pub expression: Expr,
 }
 
+/// Enumeration of the two kinds of parameter
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub enum AnyParameter<'a> {
```

Comment:
Nit: I think we should call this `AnyParameterRef` to make it clear that it is a reference and not an owned parameter (we could expose the same iterators with owned values)